.github/workflows/release.yaml

```diff
@@ -73,6 +73,26 @@ jobs:
 
           PREFIX=metal-os/${GITHUB_REF##*/} go run ./cmd/tools/generate-table > downloads.md
 
+      - name: Run Grype scan for debian
+        id: grype-scan-debian
+        uses: anchore/scan-action@v6
+        with:
+          image: ghcr.io/metal-stack/debian:latest
+          fail-build: false
+          additional-args: "-c .grype.yaml"
+          output-format: table
+
+      - name: Find issues for debian vulnerabilities
+        if: ${{ steps.grype-scan-debian.outcome == 'success' }}
+        id: debian-issues
+        uses: lee-dohm/select-matching-issues@v1
+        with:
+          query: 'label:debian-vulnerability'
+          token: ${{ github.token }}
+
+      - name: Close fixed vulnerability issues
+        run: cat ${{ steps.debian-issues.outputs.path }} | xargs gh issue close
+
```

Comment:
you can put this into pr.yaml and it will be executed during this PR

Reply:
I don't think this is the right thing to do, as it would not scan the image built in the PR, but the old one in the registry. If you have write permissions in the repo you could manually trigger the scan workflow instead.

Reply:
The scan job is not visible yet, because its on the master branch only. It is OK to let it run against the PR image to get a idea how it looks, otherwise we need to merge to master for every fix in this regard.

Lets first make it work in the PR, after we are satisfied with the results we can change it back to master and then merge this.